infrastructure/ecs_task/task_trigger_role.tf

```diff
@@ -40,7 +17,7 @@ data "aws_iam_policy_document" "task_trigger_policy" {
       "sqs:GetQueueAttributes",
```

Comment:
This policy needs splitting in half. Half of these permissions are common with lambda tasks too e.g. permissions to access the sqs queue